### ISSUE TITLE
Missing accessibility features

### ISSUE THREAD

**Issue by Allensamm** [Allensamm/mvcheck#3]
The code currently does not include any accessibility features. This could make it difficult for users with disabilities to use the application. You should consider including accessibility features, such as aria labels, to ensure that the application is accessible to all users.
